supportedBackends/local/src/main/scala/cromwell/backend/impl/local/LocalRuntimeAttributes.scala

```diff
@@ -0,0 +1,27 @@
+package cromwell.backend.impl.local
+
+import cromwell.attributes.{ContinueOnReturnCode, RuntimeAttributesValidation}
+import wdl4s.values.WdlValue
+import scalaz._
+import Scalaz._
+import lenthall.exception.MessageAggregation
+
+object LocalRuntimeAttributes extends RuntimeAttributesValidation {
+  def apply(attrs: Map[String, WdlValue]): LocalRuntimeAttributes = {
+    val continueOnReturnCode = validateContinueOnReturnCode(attrs.get("continueOnReturnCode"))
```

Comment:
Rather than discuss here, I've started a doc for this - https://docs.google.com/a/broadinstitute.com/document/d/1feRDusWXQQ2pJ03sNHTNmrrnnwL3y-vtyF1fv_RdogU/edit?usp=sharing